nmdc_automation/re_iding/scripts/re_id_tool.py

```diff
@@ -539,6 +542,14 @@ def delete_old_records(ctx, old_records_file):
     except requests.exceptions.RequestException as e:
         logging.error(f"An error occured while running: {delete_annotation_query}, response retutrned: {e}")
 
+    # write the deleted records to a tsv file
+    deleted_records_file = DATA_DIR.joinpath(f"{old_base_name}_deleted_records.tsv")
+    logging.info(f"Writing {len(deleted_records)} deleted records to {deleted_records_file}")
```

Comment:
I'd prefer this message said "deleted record identifiers" instead of "deleted records," based on what I see in the function's docstring (assuming "record" here is referring to a deleted Mongo document).

```diff
- logging.info(f"Writing {len(deleted_records)} deleted records to {deleted_records_file}")
+ logging.info(f"Writing {len(deleted_records)} deleted record identifiers to {deleted_records_file}")
```

Reply:
Done